samples/excel/85-preview-apis/comment-resolution.yaml

```diff
@@ -0,0 +1,113 @@
+order: 6
+id: excel-comment-resolution
+name: Comment resolution
+description: Resolves and reactivates a comment.
```

Comment:
Is "reactivate comment" the right term here, or do we want to use "reopen comment thread" -- to match the term used by the Excel UI? If "reopen" is the right term, then consider replacing "reactivate" with "reopen" throughout the snippet code as well.

![image](https://user-images.githubusercontent.com/3019529/65910165-0a059300-e37f-11e9-899a-2a4f000f535b.png)

Reply:
Great catch!